roles/proxy/tasks/main.yml

```diff
@@ -1,6 +1,22 @@
 ---
-- name: nginx
-  apt: name=nginx state=present
+- name: pull docker image for user containers
+  command: docker pull jupyter/systemuser
+  sudo: yes
+
+# Get static resources from the userland container
+- name: static files container
+  docker:
+    image: jupyter/systemuser
+    state: running
+    name: static-files
+    command: echo "#data-container-hack"
+    volumes:
+    - "{{ notebook_static_files }}"
```

Comment:
Yeah, totally. Just a side note. If they're ever out of sync, IPython may not behave well for clients.